packages/neos-ui-editors/src/Library/LinkInput.js

```diff
@@ -297,7 +297,7 @@ export default class LinkInput extends PureComponent {
                     value={''}
                     plainInputMode={isUri(this.state.searchTerm)}
                     onValueChange={this.handleValueChange}
-                    placeholder={this.props.i18nRegistry.translate($get('options.placeholder', this.props) || 'Neos.Neos.Ui:Main:ckeditor__toolbar__link__placeholder', 'Paste a link, or search')}
+                    placeholder={this.props.i18nRegistry.translate($get('options.placeholder', this.props))}
```

Comment:
```suggestion
                    placeholder={this.props.i18nRegistry.translate($get('options.placeholder', this.props) || 'Neos.Neos.Ui:Main:ckeditor__toolbar__link__placeholder', $get('options.placeholder', this.props) || 'Paste a link, or search')}
```

Otherwise, if no manual placeholder has been configured, the placeholder ends up empty:
![Screenshot_2024-06-03_13-13-22 LinkEditor empty](https://github.com/neos/neos-ui/assets/2522299/9def115f-7cf7-4c60-8826-65a0e519181d)

Reply:
Thank you for reviewing! I applied the code you provided, except for the chosen label. I don't see why the label should change from: `Neos.Neos:Main:content.inspector.editors.linkEditor.search` to `Neos.Neos.Ui:Main:ckeditor__toolbar__link__placeholder` only in this case.

Consequently, i adjusted `Paste a link, or search` to `Paste a link, or type to search` to better fit the default placeholder configured here:
<img width="676" alt="image" src="https://github.com/neos/neos-ui/assets/110388306/cfd5dfbd-4a8f-4146-8562-6799c30a047c">

Reply:
Well, it's not obvious, sadly, but the `<LinkInput/>` is used in two places. One is the LinkEditor (for which the `Neos.Neos:Main:content.inspector.editors.linkEditor.search` label is a fitting choice), the other is the RichTextEditorToolbar, which is why the fallback to `Neos.Neos.Ui:Main:ckeditor__toolbar__link__placeholder` exists.

But, I guess it's okay to unify this and put LinkEditor in the lead. We just gotta remember to clean up the other one :sweat_smile: